e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -25,6 +25,7 @@ import {
   updateDashboardCards,
   goToTab,
   moveDashCardToTab,
+  addTextBoxWhileEditing,
```

Comment:
This is probably a global problem affecting all undos, but posting for completeness: undo won't work after leaving dashboard edit mode.

https://github.com/metabase/metabase/assets/6830683/02bd8de8-d395-4a43-ada6-422a7a526b16

Reply:
This too has been added to M3